server/serverjs/cubefn.js

```diff
@@ -237,20 +237,23 @@ export const CSVtoCards = (csvString, carddb) => {
     'image back url': imageBackUrl,
     tags,
     notes,
-    'Color Category': colorCategory,
+    'color category': colorCategory,
     rarity,
+    'override name': overrideName,
   } of data) {
     if (name) {
       const upperSet = (set || '').toUpperCase();
       const card = {
-        name,
+        name: name,
```

Comment:
Why can't we use the shortcut notation that was used before instead of having to repeat ourselves?